src/main/java/net/kevinthedang/ollamamod/OllamaMod.java

```diff
@@ -0,0 +1,61 @@
+package net.kevinthedang.ollamamod;
+
+import com.mojang.logging.LogUtils;
+import net.minecraftforge.api.distmarker.Dist;
+import net.minecraftforge.common.MinecraftForge;
+import net.minecraftforge.event.BuildCreativeModeTabContentsEvent;
+import net.minecraftforge.event.server.ServerStartingEvent;
+import net.minecraftforge.eventbus.api.IEventBus;
+import net.minecraftforge.eventbus.api.SubscribeEvent;
+import net.minecraftforge.fml.ModLoadingContext;
+import net.minecraftforge.fml.common.Mod;
+import net.minecraftforge.fml.config.ModConfig;
+import net.minecraftforge.fml.event.lifecycle.FMLClientSetupEvent;
+import net.minecraftforge.fml.event.lifecycle.FMLCommonSetupEvent;
+import net.minecraftforge.fml.javafmlmod.FMLJavaModLoadingContext;
+import org.slf4j.Logger;
+
+// The value here should match an entry in the META-INF/mods.toml file
+@Mod(OllamaMod.MOD_ID)
+public class OllamaMod {
+    // Define mod id in a common place for everything to reference
+    public static final String MOD_ID = "ollamamod";
+    // Directly reference a slf4j logger
+    private static final Logger LOGGER = LogUtils.getLogger();
```

Comment:
The slf4j docs/info can be found somewhere (are referenced, if needed), correct?

Reply:
Will have to look into this, I following a short guide.

Reply:
To what I know now, it is an imported gradle library type of logger.

It can be found on `line 16` as 
```java
import org.slf4j.Logger;
```

I will add to the comment that it "references" a slf4j logger.